Libraries/LibWeb/HTML/HTMLObjectElement.cpp

```diff
@@ -92,6 +92,13 @@ bool HTMLObjectElement::will_validate()
     return false;
 }
 
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#dom-cva-validationmessage
+String HTMLObjectElement::validation_message() const
+{
+    // Returns the empty string, as object elements are not subject to constraint validation.
+    return {};
+}
```

Comment:
I don't think this method is necessary because `FormAssociatedElement::validation_message()` would also return the empty string.